### ISSUE TITLE
Roadmap tooltip needs to work on click, not hover

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#356]
# What happened

User research clearly demonstrates the hover for tooltips doesn't work very well. Users cannot see all the information, and thre is no way to scroll down without losing the tooltip entirely.

# What I expected to happen

Tooltips need to stay open as long as the user needs it.

**Comment by imperator99** [comment]
Resolved by https://github.com/govau/dta-gov-au/releases/tag/1.0.4.
